src/app/components/core/south/add-service-wizard/add-service-wizard.component.html

```diff
@@ -30,35 +30,50 @@
             </div>
             <div class="field-body">
               <div class="select is-multiple">
-                <select multiple name="type" #selectedPlugin formControlName="plugin" (change)="getDescription(selectedPlugin.value)"
-                  required>
+                <select multiple name="type" #selectedPlugin formControlName="plugin"
+                  (change)="getDescription(selectedPlugin.value)" required>
                   <option *ngFor="let p of plugins" [value]="p.name"> {{ p.name }}</option>
                 </select>
               </div>
               &nbsp;
-              <small *ngIf="selectedPluginDescription && isValidPlugin && isSinglePlugin" class="help description level-left">{{
+              <small *ngIf="selectedPluginDescription && isValidPlugin && isSinglePlugin"
+                class="help description level-left">{{
                 selectedPluginDescription }}</small>
               <small *ngIf="!isValidPlugin" class="help is-danger level-left">* Please select a valid plugin</small>
               <small *ngIf="!isSinglePlugin" class="help is-danger level-left">* Please select a single plugin</small>
             </div>
           </div>
+          <div *ngIf="plugins.length > 0" class="field is-horizontal form-group is-right">
```

Comment:
What is this length check for?

Reply:
when no plugin is installed. Just show link button next to the message `No south plugin installed. Install from _available plugins_`